### PR TITLE
Clean doc & update changes from the root testng project

### PR DESCRIPTION
**Pull request by juherr**

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -1,3 +1,7 @@
-Redirect 301 /eclipse-beta https://dl.bintray.com/testng-team/testng-eclipse/
-AddHandler php4-script .php
-Redirect temp /dtd http://testng.org/testng-1.0.dtd.html
+Redirect 301 /eclipse-beta http://dl.bintray.com/testng-team/testng-eclipse/
+Redirect 301 /testng-1.0.dtd http://raw.githubusercontent.com/cbeust/testng/master/src/main/resources/testng-1.0.dtd
+Redirect 301 /index.html http://testng.org/doc/
+Redirect 301 /licence/index.html http://testng.org/doc/licence/index.html
```
Comment (cbeust) [inline .htaccess]: "licence" with a 'c'?

**Comment by juherr** [comment]
@cbeust Done! The git exercise was fun ;)

**Comment by cbeust** [comment]
Thanks for keeping me sane, @juherr  :)

**Comment by juherr** [comment]
@cbeust Let me know when it is online. I will check if everything is working fine.

**Comment by missedone** [comment]
I see this on Chrome on Mac 10.12:
screenshot at feb 25 17-29-41

**Comment by cbeust** [comment]
I don't see the error. How about you @juherr? (note: I haven't deployed
your latest change to the doc)

-- 
Cédric

**Comment by missedone** [comment]
looks like I did not see this any more ...

**Pull request merged**